test/testenv/verificationutils.go

```diff
@@ -252,23 +252,68 @@ func VerifyServiceAccountConfiguredOnPod(deployment *Deployment, ns string, podN
 		}
 		logf.Log.Info("Service Account on Pod", "FOUND", restResponse.Spec.ServiceAccount, "EXPECTED", serviceAccount)
 		return strings.Contains(serviceAccount, restResponse.Spec.ServiceAccount)
-	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(true))
+	}, ConsistentDuration, ConsistentPollInterval).Should(gomega.Equal(true))
 }
 
 // VerifyIndexFoundOnPod verify index found on a given POD
 func VerifyIndexFoundOnPod(deployment *Deployment, podName string, indexName string) {
-	gomega.Eventually(func() bool {
-		indexFound := GetIndexOnPod(deployment, podName, indexName)
+	gomega.Consistently(func() bool {
+		indexFound, _ := GetIndexOnPod(deployment, podName, indexName)
 		logf.Log.Info("Checking status of index on pod", "PODNAME", podName, "INDEX NAME", indexName, "STATUS", indexFound)
 		return indexFound
-	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(true))
+	}, ConsistentDuration, ConsistentPollInterval).Should(gomega.Equal(true))
+}
+
+// VerifyIndexConfigsMatch verify index specific config
+func VerifyIndexConfigsMatch(deployment *Deployment, podName string, indexName string, maxGlobalDataSizeMB int, maxGlobalRawDataSizeMB int) {
+	gomega.Consistently(func() bool {
+		indexFound, data := GetIndexOnPod(deployment, podName, indexName)
+		logf.Log.Info("Checking status of index on pod", "PODNAME", podName, "INDEX NAME", indexName, "STATUS", indexFound)
+		if indexFound == true {
+			if data.Content.MaxGlobalDataSizeMB == maxGlobalDataSizeMB && data.Content.MaxGlobalRawDataSizeMB == int(maxGlobalRawDataSizeMB) {
```

Comment:
nit: wondering, why we need to int(), probably, you mean for lvaue?. If so, for both conditions?